packages/react-ui-components/src/stories/Components/Checkbox.stories.tsx

```diff
@@ -0,0 +1,46 @@
+import { ComponentStory } from '@storybook/react';
+import React, { useCallback, useState } from 'react';
+import { KvCheckbox } from '../../components';
+
+export default {
+	title: 'Inputs/Checkbox',
+	component: 'kv-checkbox',
+	parameters: {
+		notes: require('@ui-notes/checkbox/readme.md')
+	}
+};
+
+KvCheckbox.displayName = 'Checkbox';
+
+const CheckboxTemplate: ComponentStory<typeof KvCheckbox> = args => {
```

Comment:
Couldn't we do `const CheckboxTemplate: ComponentStory<typeof KvCheckbox> = ({ checked: initialChecked, ...otherArgs }) =>{`  ?